certbot_dns_desec/dns_desec.py

```diff
@@ -86,8 +87,27 @@ def __init__(self, endpoint, token):
         self.session.headers["Authorization"] = f"Token {token}"
         self.session.headers["Content-Type"] = "application/json"
 
+    @staticmethod
+    def desec_request(method, **kwargs):
+        response: requests.Response = method(**kwargs)
```

Comment:
From which Python version on is this legal? (We may want to be more backwards-compatible.)

Reply:
[Docs say 3.5](https://docs.python.org/3/library/typing.html), I've tested [`x: int = 4`](https://xkcd.com/221/) successfully with 3.6.